scripts/keychain-cli.js

```diff
@@ -994,7 +998,10 @@ async function run() {
         await keymaster.start({
             gatekeeper: gatekeeper_sdk,
             wallet: db_wallet,
-            cipher
+            cipher,
```

Comment:
If KC_ENCRYPTED_WALLET is true, pass in the encrypted wallet here after initialing it with the pass phrase vars.